sycl/include/CL/sycl/detail/pi.h

```diff
@@ -640,6 +640,14 @@ pi_result piProgramRelease(pi_program program);
 //
 // Kernel
 //
+
+typedef enum {
+  PI_USM_INDIRECT_ACCESS,
```

Comment:
Please describe the meaning of the info not borrowed from OpenCL
Remember to give its type (pi_bool)

Reply:
It's a tag, not a type.  There's just no corresponding CL value.  I don't care what the value is, only that it's unique in the enum so I can compare it.

Reply:
I understand that, but I am asking to document the kind and type of the information being set by this tag.

Reply:
Aka what the tag means and what sort of thing would be passed as the value in the function call?

Reply:
Yes, thanks. Please do say it expects a pi_bool value.